package.json

```diff
@@ -5,6 +5,11 @@
   "main": "src/index.js",
   "author": "ubio",
   "license": "MIT",
+  "files": [
```

Comment:
Nice :+1:

Reply:
Is it worth moving the schema up a level out of the `src` directory? That'd make this nice and easy, and avoid potential issues in the future.